grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -751,6 +751,45 @@ class ProductAvailabilityService {
         return data
     }
 
+    List<AvailableItem> getAvailableItems(Location bin, boolean excludeZeroQuantity = true) {
+        log.info("getItemsInBin: bin=${bin}")
+
+        if (!bin) {
+            return []
+        }
+
+        String quantityCondition = excludeZeroQuantity ? "and pa.quantityOnHand > 0" : ""
```

Comment:
I think we should always exclude zero QoH, not sure if this part is needed

Reply:
Corrected